api/params.go

```diff
@@ -450,6 +451,9 @@ func (p *Params) SetDefaults(gitSource, gitOwner, gitName, appLabel, buildVersio
 	if p.Request.Timeout == "" {
 		p.Request.Timeout = "60s"
 	}
+	if p.Request.KeepaliveTimeout == "" {
+		p.Request.KeepaliveTimeout = "20s"
```

Comment:
I think the default keepalive_timeout is 75s. Why are we lowering it to 20s? Also i think Nginx closes keepalive connections when the worker_connections limit is reached. Is there a specific usecase we are trying to solve with this PR?

Reply:
This is only the default value. Because our upstream timeout of ingress is 15 seconds, whether it is necessary to meet our use scenarios, the smaller the better.